src/gluonts/ev/metrics.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from typing import Collection, Optional, Union, Callable
+
+from dataclasses import dataclass
+
+import numpy as np
+
+from .api import PointMetric, LocalMetric
+
+
+# --- POINT METRICS ---
+
+
+class AbsTarget(PointMetric):
+    name: str = "abs_target"
+
+    def get(self, input_data, label, forecast, metrics) -> np.ndarray:
+        return np.abs(label["target"])
+
+
+class Error(PointMetric):
+    name: str = "error"
+
+    def get(self, input_data, label, forecast, metrics) -> np.ndarray:
+        return label["target"] - forecast.median
+
+
+class AbsError(PointMetric):
+    name: str = "abs_error"
+    dependencies = [Error()]
+
+    def get(self, input_data, label, forecast, metrics) -> np.ndarray:
```

Comment:
Metrics and input data are always provided by the `Evaluator` "just to be sure", since _some_ metrics it. This is done internally, though. The user just passes a `TestDataset` and a `Forecast` for evaluation. Having a lot of unused parameters is a bit ugly, I agree, but it shouldn't hurt performance.